Examples/Widgets/CRM/src/components/Summary.ts

```diff
@@ -0,0 +1,127 @@
+    return {
+      "compact": this.compact
+    }
+  }
+
+  static get styles() {
+    return styles;
+  }
+
+  render() {
+    return html`
+      <div class="summary-container" part="summary">
+        <section class="cust-info-header">
+          <md-avatar
+            src=${ifDefined(this.customerData?.picture)}
+            size="56"
+          ></md-avatar>
+          <h1>${this.customerData?.name}</h1>
+          <div class="age-gender">
+            ${this.customerData?.age} years old, ${this.customerData?.gender}
+          </div>
+          <div class="phone">
+            <md-icon name="icon-audio-call_14" size="10"></md-icon>
+            ${this.customerData?.phone}
+          </div>
+        </section>
+
+        <md-badge small color="green" pill="false"
+          >MRN# ${this.customerData?.MRN}</md-badge
+        >
+        <table class="${classMap(this.resizeClassMap)}">
+          <tr>
+            <td class="title">
+              Date of Birth
+            </td>
+            <td class="value">
+              ${this.customerData?.DOB}
+            </td>
+          </tr>
+          <tr>
+            <td class="title">
+              Address
+            </td>
+            <td class="value">
+              ${this.customerData?.address}
+            </td>
+          </tr>
+          <tr>
+            <td class="title">
+              Insurance
+            </td>
+            <td class="value">
+              <span>${this.customerData?.insurance.provider}</span><br />
+              <span>${this.customerData?.insurance.planName}</span><br />
+              <span>${this.customerData?.insurance.planNumber}</span><br />
+              <span>${this.customerData?.insurance.groupNumber}</span><br />
+              <span>${this.customerData?.insurance.memberID}</span>
+            </td>
+          </tr>
+          <tr>
+            <td class="title">
+              Next of Kin
+            </td>
+            <td class="value">
+              <span
+                >${this.customerData?.nextToKin.name}
+                ${this.customerData?.nextToKin["phone number"]}</span
+              >
+            </td>
+          </tr>
+          <tr>
+            <td class="title">
+              Conditions
+            </td>
+            <td class="value">
+              ${this.customerData?.conditions.map(
+                condition =>
+                  html`
+                    <span>${condition}</span><br />
+                  `
+              )}
+            </td>
+          </tr>
+          <tr>
+            <td class="title">
+              Allergies
+            </td>
+            <td class="value">
+              ${this.customerData?.allergies.map((allergy, i, arr) => {
+                return html`
+                  <span>${allergy}</span>${i < arr.length - 1 ? " | " : nothing}
+                `;
+              })}
+            </td>
+          </tr>
+        </table>
+
+        <md-button type="button" variant="primary" size="32"
```

Comment:
did you ask Lily what it was supposed to do? if it does not do anything, remove it for now